questdb-rs/src/ingress/mod.rs

```diff
@@ -26,12 +26,17 @@
 //!
 //! The `ingress` module implements QuestDB's variant of the
 //! [InfluxDB Line Protocol](https://questdb.io/docs/reference/api/ilp/overview/)
-//! (ILP) over TCP.
+//! (ILP) over HTTP and TCP. We recommend using HTTP, and consider TCP as a
+//! legacy protocol variant.
 //!
 //! To get started:
-//!   * Connect to QuestDB by creating a [`Sender`] object, usually via [`Sender::from_conf`].
-//!   * Populate a [`Buffer`] with one or more rows of data.
-//!   * Send the buffer via the Sender's [`flush`](Sender::flush) method.
+//!   * Make sure you have the dependency with the HTTP feature enabled:
+//!
+//! ```cargo add questdb-rs --features ilp-over-http```
+//!
+//!   * Use [`Sender::from_conf()`] to get the [`Sender`] object
+//!   * Populate a [`Buffer`] with one or more rows of data
+//!   * Send the buffer using [`sender.flush()`](Sender::flush)
```

Comment:
I think it's OK to revert this doc change and simply make `ilp-over-http` default as you earlier suggested.